### ISSUE TITLE
Agda --doublecheck cannot resolve level

### ISSUE THREAD

**Issue by EdNutting** [agda/agda-stdlib#1051]
Hi,

https://github.com/agda/agda-stdlib/blob/aa7c23b7561b99e15bb7572e5ddc64cf1e1deff8/src/Relation/Binary/Morphism/Definitions.agda#L30

Loading this module with `agda --safe --doublecheck ...` causes it to complain that it can't work out the output level. Not sure if this is expected/desired or not. The fix seems to be to replace `Set _` with `Set (a ⊔ ℓ₁ ⊔ ℓ₂)`.

Cheers,
Ed

**Comment by gallais** [comment]
Should this issue be moved upstream? (ping @jespercockx)

**Comment by jespercockx** [comment]
Yes please, you can report it as a bug on github,com/agda/agda. If possible, it would be very helpful to have a standalone example (i.e. not dependent on the stdlib) but an example that does depend on it is better than nothing at all.
